hoprd/rest-api/src/account.rs

```diff
@@ -77,6 +77,7 @@ pub(crate) struct AccountBalancesResponse {
         responses(
             (status = 200, description = "The node's HOPR and Safe balances", body = AccountBalancesResponse),
             (status = 401, description = "Invalid authorization token.", body = ApiError),
+            (status = 412, description = "The node is not ready."),
```

Comment:
Why does this not have the `body = ApiError`?

Reply:
True. Removed.